stubs/projects/index.json

```diff
@@ -0,0 +1,84 @@
+[
+  {
+    "id": 12,
+    "description": "",
+    "default_branch": "master",
+    "tag_list": [],
+    "public": false,
+    "archived": false,
+    "visibility_level": 0,
+    "ssh_url_to_repo": "git@gitlab.example.com:group/project",
+    "http_url_to_repo": "https://gitlab.example.com/group/project.git",
+    "web_url": "https://gitlab.example.com/group/project",
+    "name": "project",
+    "name_with_namespace": "group / project",
+    "path": "project",
+    "path_with_namespace": "group/project",
+    "issues_enabled": true,
+    "merge_requests_enabled": true,
+    "wiki_enabled": false,
+    "snippets_enabled": false,
+    "created_at": "2015-10-26T14:32:44.103+01:00",
+    "last_activity_at": "2015-11-01T22:07:47.456+01:00",
+    "creator_id": 1,
+    "namespace": {
+      "id": 7,
+      "name": "group",
+      "path": "group",
+      "owner_id": null,
+      "created_at": "2015-10-26T13:36:40.147+01:00",
+      "updated_at": "2015-10-27T08:57:55.199+01:00",
+      "description": "group desc",
+      "avatar": {
+        "url": "/uploads/group/avatar/7/group-logo.png"
+      }
+    },
+    "avatar_url": "https://gitlab.example.com/uploads/project/avatar/12/group-logo.png",
+    "star_count": 0,
+    "forks_count": 0
+  },
+  {
+    "id": 11,
+    "description": "",
+    "default_branch": "master",
+    "tag_list": [],
+    "public": false,
+    "archived": false,
+    "visibility_level": 0,
+    "ssh_url_to_repo": "git@gitlab.example.com:johndoe/project2.git",
+    "http_url_to_repo": "https://gitlab.example.com/johndoe/project2.git",
+    "web_url": "https://gitlab.example.com/johndoe/project2",
+    "owner": {
+      "name": "John Doe",
+      "username": "johndoe",
+      "id": 1,
+      "state": "active",
+      "avatar_url": "https://secure.gravatar.com/avatar/123",
+      "web_url": "https://gitlab.example.com/u/johndoe"
+    },
+    "name": "project2",
+    "name_with_namespace": "John Doe / project2",
+    "path": "project2",
+    "path_with_namespace": "johndoe/project2",
+    "issues_enabled": true,
+    "merge_requests_enabled": true,
+    "wiki_enabled": true,
+    "snippets_enabled": true,
+    "created_at": "2015-03-18T19:33:20.166+01:00",
+    "last_activity_at": "2015-11-01T19:48:32.793+01:00",
+    "creator_id": 1,
+    "namespace": {
+      "id": 1,
+      "name": "John Doe",
+      "path": "johndoe",
+      "owner_id": 1,
+      "created_at": "2014-01-05T22:01:51.000+01:00",
+      "updated_at": "2014-01-05T22:51:08.000+01:00",
+      "description": "",
+      "avatar": null
+    },
+    "avatar_url": null,
+    "star_count": 0,
+    "forks_count": 0
+  }
+]
```

Comment:
Same thing, please end the file with a newline to avoid diffs on this in later commits.